ArjunaJTA/jta/tests/classes/com/hp/mwtests/ts/jta/tools/ObjStoreBrowserTest.java

```diff
@@ -49,7 +57,7 @@
  * provide a better separation between public and internal classes.
  */
 @Deprecated // in order to provide a better separation between public and internal classes.
-class ExtendedFailureXAResource extends FailureXAResource {
+class ExtendedFailureXAResource extends FailureXAResource implements XAResourceWrapper {
```

Comment:
This implementation is redundant since you have just added it to FailureXAResource.